### PR TITLE
Review

### PR DESCRIPTION
**Pull request by cramraj8**
This is for review purpose.

### CODE REVIEW

File: reg.py [hunk 1]
```diff
@@ -29,8 +30,21 @@
        optimizer = tf.train.AdamOptimizer()             ===> Gives definite values
 '''
 
-def read_dataset():
+def read_dataset(name=None):
+    """
+    read_dataset reads the data set, interpret, and return feature, label extracted data set.
```
Comment (safooray) [inline reg.py:35]: -Lines should not be longer than 80 characters. Fix this everywhere.
-The one line summary doesn't make sense.

File: reg.py [hunk 2]
```diff
@@ -29,8 +30,21 @@
...
 
+  Args:
+    Nothing
+
+  Returns:
+    `Nump array`, extracted feature columns and label column.
```
Comment (safooray) [inline reg.py:41]: Numpy

File: reg.py [hunk 3]
```diff
@@ -29,8 +30,21 @@
...
+
+  Example:
+    >>> read_dataset()
+    ( [[2.3, 2.4, 6.5],[2.3, 5.4,3.3]], [12, 82] )
+
+    """
     D = sio.loadmat('Brain_Integ.mat')
```
Comment (safooray) [inline reg.py:48]: I remember we talked about using .csv format. Change the reader to read from .csv. Alternatively you can have read_csv and read_mat functions.

File: reg.py [hunk 4]
```diff
@@ -50,23 +64,26 @@ def read_dataset():
 
 
 
-# Model Creation
-def multilayer_perceptron(x, weights, biases):
```
Comment (safooray) [inline reg.py:54]: We talked about using Slim layers. Please switch to it.

File: reg.py [hunk 5]
```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import, division, print_function
```
Comment (safooray) [inline reg.py:1]: choose a descriptive file name.

File: reg.py [hunk 6]
```diff
@@ -50,23 +64,26 @@ def read_dataset():
...
-    # Hidden layers with RELU activation
-    layer_1 = tf.add(tf.matmul(x, weights['h1']), biases['b1'])
-    layer_1 = tf.nn.relu(layer_1)
-
-    layer_2 = tf.add(tf.matmul(layer_1, weights['h2']), biases['b2'])
-    layer_2 = tf.nn.relu(layer_2)
-
-    layer_3 = tf.add(tf.matmul(layer_2, weights['h3']), biases['b3'])
-    layer_3 = tf.nn.relu(layer_3)
+X, Y = read_dataset()
```
Comment (safooray) [inline reg.py:67]: use lower case for variable names. use descriptive variable names.

File: reg.py [hunk 7]
```diff
@@ -50,23 +64,26 @@ def read_dataset():
...
+X, Y = shuffle(X, Y, random_state=1)
+X_train, X_test, Y_train, Y_test = train_test_split(X, Y, test_size=0.20, random_state=420)
 
-    layer_4 = tf.add(tf.matmul(layer_3, weights['h4']), biases['b4'])
-    layer_4 = tf.nn.relu(layer_4)
+total_len = X_train.shape[0]            # Total Samples
+# =========== Parameters ============
+learning_rate = 0.01                    # Learning rate for parameter updates
```
Comment (safooray) [inline reg.py:73]: Constants should be upper case and at the top of the module.

File: reg.py [hunk 8]
```diff
@@ -178,6 +189,13 @@ def multilayer_perceptron(x, weights, biases):
     print (" *********************************************** ")
```
Comment (safooray) [inline reg.py:191]: -Remove prints unless information is important. 
-Run the session only once and get all the values you need.

File: reg.py [hunk 9]
```diff
@@ -178,6 +189,13 @@ def multilayer_perceptron(x, weights, biases):
...
 
 
-
+    # plt.plot(X_train[:, 0], sess.run(pred, feed_dict={x: X_train}), label='Fitted line')     # plotting the fitted line; y = mx + c
+    plt.plot(Y_train, sess.run(pred, feed_dict={x: X_train}), 'ro', label='Correlation of Original with Predicted data')                         # for marking points in space of labels and features
```
Comment (safooray) [inline reg.py:193]: You don't need to do plotting in the context of the session. Take plotting out of the session.

**Review by safooray — COMMENTED** [review]
Ramraj, please address the first round of comments.

File: reg.py (hunk 5 above)
Comment (cramraj8) [reply reg.py:1]: by descriptive file name, you mean change from reg.py to something.py ?

File: reg.py (hunk 4 above)
Comment (cramraj8) [reply reg.py:54]: Ya. I will switch to Slim. Initially, I wanted to get some practice on raw_tensorflow. So now I can change it to Slim.

File: reg.py (hunk 5 above)
Comment (safooray) [reply reg.py:1]: to like model_train.py or something. Later we will move the model to a separate module called model.py. And we will have a model_eval.py module as well.

**Pull request closed without merging**